salt/states/ssh_known_hosts.py

```diff
@@ -27,6 +27,25 @@
 import salt.utils
 from salt.exceptions import CommandNotFoundError
 
+# Define the state's virtual name
+__virtualname__ = 'ssh_known_hosts'
+
+
+def __virtual__():
+    '''
+    Does not work on Windows, requires ssh module functions
+    '''
+    if salt.utils.is_windows():
+        return False, 'ssh_known_hosts: Does not support Windows'
+
+    if 'ssh.set_known_host' not in __salt__ \
```

Comment:
This is a very expensive call and seems unnecessary. The `ssh` module is always available on non-Windows platforms. This just duplicates the check for Windows above. Could you clarify why you believe this is necessary, please?